blazor/datagrid/cell-edit-types.md

```diff
@@ -716,4 +716,270 @@ You can able to render SfMultiSelect component in EditTemplate. In the below sam
 ```
 
 In the following image, **SfMultiSelect** component is rendered with **EditTemplate** in ChosenItems column
-![Blazor DataGrid with Editing in Custom MultiSelect](./images/blazor-datagrid-editing-in-custom-multiselect.png)
\ No newline at end of file
+![Blazor DataGrid with Editing in Custom MultiSelect](./images/blazor-datagrid-editing-in-custom-multiselect.png)
+
+### Using RichTextEditor in EditTemplate
+
+You can render the **SfRichTextEditor** component in [EditTemplate](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.GridColumn.html#Syncfusion_Blazor_Grids_GridColumn_EditTemplate). To display the RTE text in the Grid, you can disable the [DisableHtmlEncode](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.GridColumn.html#Syncfusion_Blazor_Grids_GridColumn_DisableHtmlEncode) property of the [GridColumn](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.GridColumn.html). In the following sample, the **SfRichTextEditor** component is rendered in [EditTemplate](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.GridColumn.html#Syncfusion_Blazor_Grids_GridColumn_EditTemplate) for the Customer ID column.
```

Comment:
I already provided comments like don't link API reference for same API many times in the single para or corresponding topic. Here EDitTemplate property is linked with API reference for multiple times